Tasks/Mardoyan/Task3/Task3_10.py

```diff
@@ -0,0 +1,12 @@
+print('Hello!I can return the largest prime number in your list.')
+x = input('Input you list of numbers(Throught "space")\n')
+prime_list = []
+max = 0
+x = x.split(' ')
+for i in x:
+    if (int(i) % 2 != 0 and int(i) % 3 != 0) or ( int(i) == 2 or int(i) == 3):
```

Comment:
only 2 and 3 and no other numbers? not good enough

Reply:
I don't understand enough why it is incorrectly.The program works great.
I could solve it through a nested loop, but in this version it works